frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx

```diff
@@ -312,7 +313,11 @@ export const LocalVolumeSetHeader = () => {
 
   return (
     <>
-      <h1 className="co-create-operand__header-text">{t('lso-plugin~Local Volume Set')}</h1>
+      <TextContent>
+        <Text component={TextVariants.h3} className="ocs-install-wizard__h3">
```

Comment:
We dont want to use ocs class in lso page. I would suggest to make it a prop.
```suggestion
        <Text component={TextVariants.h3} className={className}>
```
Also, this will do the change in lso page form as well and for lso page, we still want to use `h1`, perhaps a prop `component` would be good to pass from ocs to this component.

Reply:
This is a generic component, hence we must add props to it , rather than making a direct change.

Reply:
@afreen23 I will make the changes. Thanks :+1: